components/tooltip/tooltip.js

```diff
@@ -366,7 +366,7 @@ class Tooltip extends RtlMixin(LitElement) {
 
 		this.announced = false;
 		this.closeOnClick = false;
-		this.delay = 0;
+		this.delay = 300;
```

Comment:
Jeff and I discussed this a fair amount and he ultimately decided we should update the default delay for all tooltips to make them less jarring.  There are a couple of places that provide their own delay... most usages rely on the default.